docs/getting_started.rst

```diff
@@ -51,12 +54,34 @@ Installation
   pip install https://github.com/RSGInc/populationsim/zipball/master
 
 
+.. _anaconda_notes :
+
+Python 2 or 3?
+~~~~~~~~~~~~~~~
+
+.. note::
+
+  PopulationSim is a 64bit Python 2 or 3 library that uses a number of packages from the
+  scientific Python ecosystem, most notably `pandas <http://pandas.pydata.org>`__
+  and `numpy <http://numpy.org>`__. It relies heavily on the
+  `ActivitySim <https://activitysim.github.io>`__ package. Both ActivitySim and PopulationSim
+  currently support Python 2, but Python 2 will be `retired <https://pythonclock.org/>`__ at the
+  end of 2019 so Python 3 is recommended.
+
+  The recommended way to get your own scientific Python installation is to
+  install 64 bit Anaconda, which contains many of the libraries upon which
+  ActivitySim depends + some handy Python installation management tools.
+
+  For more information on Anaconda and ActivitySim, see ActivitySim's `getting started
+  <https://activitysim.github.io/activitysim/gettingstarted.html#anaconda>`__ guide.
+
+
 Run Examples
 ------------
 
   * Before running examples, ensure that Anaconda Python, dependent libraries and PopulationSim package have been installed.
 
-  * Download and unzip the `example setups <https://github.com/RSGInc/populationSim_resources/raw/master/example_setup/PopulationSimExampleSetUps.7z>`_ to a folder on your computer. It does not have to be the same directory as your Anaconda or PopulationSim install.
+  * Download and unzip the `example setups <https://github.com/RSGInc/populationSim_resources/raw/master/example_setup/PopulationSimExampleSetUpsPython3.zip>`_ to a folder on your computer. It does not have to be the same directory as your Anaconda or PopulationSim install.
```

Comment:
this folder has already been updated with the latest Python3 compatible tests

Reply:
This looks good